internal/provider/resource_port_cloud_router_connection.go

```diff
@@ -181,3 +193,23 @@ func extractOwnedPortConn(d *schema.ResourceData) packetfabric.CustomerOwnedPort
 	}
 	return ownedPort
 }
+
+func splitCloudRouterCircuitIdString(data string) (CloudRouterCircuitIdData, error) {
```

Comment:
Could you move this function to this file `resource_cloud_router_connection.go` as it will be eventually used by all the cloud router connection terraform resources?